test/integration/hotrestart_test.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [[ `uname` != "Linux" ]]
```

Comment:
I think there's probably a way this test could be completely disabled with a Bazel rule instead.

@htuch